pynestml/codegeneration/nest_codegenerator.py

```diff
@@ -136,24 +197,24 @@ def generate_module_code(self, neurons: List[ASTNeuron]) -> None:
         if not os.path.exists(FrontendConfiguration.get_target_path()):
             os.makedirs(FrontendConfiguration.get_target_path())
 
-        with open(str(os.path.join(FrontendConfiguration.get_target_path(),
-                                   FrontendConfiguration.get_module_name())) + '.h', 'w+') as f:
-            f.write(str(self._template_module_header.render(namespace)))
-
-        with open(str(os.path.join(FrontendConfiguration.get_target_path(),
-                                   FrontendConfiguration.get_module_name())) + '.cpp', 'w+') as f:
-            f.write(str(self._template_module_class.render(namespace)))
-
-        with open(str(os.path.join(FrontendConfiguration.get_target_path(),
-                                   'CMakeLists')) + '.txt', 'w+') as f:
-            f.write(str(self._template_cmakelists.render(namespace)))
-
         if not os.path.isdir(os.path.realpath(os.path.join(FrontendConfiguration.get_target_path(), 'sli'))):
```

Comment:
Is this still necessary?

Reply:
Removed `sli` file and directory from `CMakeLists.txt`